unit-tests/shared/src/test/scala/javalib/net/InetAddressTest.scala

```diff
@@ -40,6 +42,12 @@ class InetAddressTest {
     assertFalse(all == null)
     assertTrue(all.length >= 1)
 
+    if (!Platform.isWindows) {
```

Comment:
Not to hold up the PR but may I ask why the test
is not being run on Windows?  That is, beyond "because
it fails there (dolt!)".

Windows has a `getnameinfo` call, so I would expect
`getCannonicalName` to succeed there.

`NetDbTest.scala` has no test case for `getnameinfo`. The
test cases that are there are ` if (!isWindows) {`

Pending additional info here, I suggest a one line
comment above line 45, leaving at least a clue as
to why the test is not being run on Windows.  Then
I can create a new Issue, referencing this PR,
requesting a Test for `getnameinfo` and wondering if it
 works on Windows.

That get the business of this PR done and delay either
of us getting sucked further into the network tar pit.

Reply:
Here's the stacktrace.
```
[error] Test javalib.net.InetAddressTest.getAllByName failed: java.lang.AssertionError: null, took 0.0 sec
[error]     at javalib.net.InetAddressTest$$anonfun$getAllByName$2.apply(InetAddressTest.scala:45)
[error]     at javalib.net.InetAddressTest$$anonfun$getAllByName$2.apply(InetAddressTest.scala:44)
[error]     at scala.collection.TraversableLike$WithFilter$$anonfun$foreach$1.apply(TraversableLike.scala:733)
[error]     at scala.collection.IndexedSeqOptimized$class.foreach(IndexedSeqOptimized.scala:33)
[error]     at scala.collection.mutable.ArrayOps$ofRef.foreach(ArrayOps.scala:186)
[error]     at scala.collection.TraversableLike$WithFilter.foreach(TraversableLike.scala:732)
[error]     at javalib.net.InetAddressTest.getAllByName(InetAddressTest.scala:44)
```

Reply:
In fact, that is a JVM stacktrace. The test does not pass on Windows JVM.

Reply:
OK, I will have to run the test on Windows to see what the error is there.   

Seems to me that at the very least there is a design flaw  in `InetAddressTest`

Not my biggest problem this month, but I am trying to accumulate a backlog of
Issues for the long, cold Winter ahead, when I have nothing to do but chase
low level causes of displeasure.